### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CuriouslyCurious**
I've updated the `README.md` with all the changes relevant I could think of, now that you've started using a build script. Feel free to change anything I got wrong.

There's some dependency I'm missing for the `./deploy.sh` script, so I haven't added that to the prerequisites.

Also, note to self: don't try to amend a merge, it looks super weird.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,77 +2,71 @@
...
-   https://info.qt.io/download-qt-for-application-development
-   
-   NOTE: The latest kit that was verified to build NESICIDE properly with no errors is Qt 5.6.2. Please do not 
-         use a kit later than this, or if you do, please create a pull request with necessary changes. Known issues:
-         
-         . C++11 support required
+0. Install `git`
 
-1. Install git
+1. Install `Qt 5.6.2` [https://download.qt.io/official_releases/qt/5.6/5.6.2/](https://download.qt.io/official_releases/qt/5.6/5.6.2/) as it was the latest kit to be verified to build NESICIDE properly with not errors. Please do not use a kit later than this, or if you do, please create a pull request with nexessary changes.
```
Comment (christopherpow) [inline README.md]: nexessary?

File: README.md [hunk 2]
```diff
@@ -2,77 +2,71 @@
...
 Building
 ========
-0. Extract dependencies into top level (nesicide/) folder:
+To build the project follow the instructions below:
 
-   tar xjvf nesicide-deps.tar.bz2
+NOTE: The build process takes a while, so grab a snack while you wait. ;)
 
-1. Build nesicide using the build scripts:
+## Linux
```
Comment (christopherpow) [inline README.md]: Is there some significance between '###' above and '##' here?

File: README.md [hunk 3]
```diff
@@ -2,77 +2,71 @@
...
-   b. NES Emulator (stand alone package of the emulator used in the IDE):
-
-      apps/nes-emulator/release/
+Running
+=======
+The `deploy.sh` creates executables for all platforms at the following locations:
 
-   c. FamiTracker:
+* NESICIDE:
+`apps/ide/release/nesicide`
```
Comment (christopherpow) [inline README.md]: The deploy script will soon be updated to push releases to github as soon as I can figure all that out.

File: README.md (hunk 1 above)
Comment (CuriouslyCurious) [reply README.md]: You still of course need to install the right QT version but the notice about the release might not be needed.

File: README.md (hunk 3 above)
Comment (CuriouslyCurious) [reply README.md]: Should that be excluded then?

File: README.md (hunk 2 above)
Comment (CuriouslyCurious) [reply README.md]: No, just a typo.

File: README.md (hunk 3 above)
Comment (christopherpow) [reply README.md]: Nah I'll update it once I get that part done. Just a note.

File: README.md (hunk 1 above)
Comment (CuriouslyCurious) [reply README.md]: Actually, should this be excluded altogether and replaced with an appropriate package manager command for the latest version of QT?

Comment (CuriouslyCurious) [reply README.md]: Oh, I see. Haha. Spelling errors are hard to spot at times. :D

**Comment by CuriouslyCurious** [comment]
There we go, I think.

Once you think you are done with the `./deploy.sh` script, just add whatever it depends on to the list of prerequisites.

**Comment by christopherpow** [comment]
Agreed. I will do that.

**Pull request merged**